requirements/install.txt

```diff
@@ -0,0 +1,11 @@
+#use specific pip version for dependencies to work nicely
+pip==22.0.4
```

Comment:
Legacy code, i don't know either. what i do know is this was done to fix an issue. In `tasks.py` there's the line 
``` 
print('--> Fixing a pip issue, failing to import `BAR_TYPES` from `pip._internal.cli.progress_bars`')
    c.run(PYTHON_EXE + ' -m pip install --upgrade pip==22.0.4')
```

One of the branches that has started the refactor (and we're hitting bottlenecks, which is why it isn't part of the stable release) updates all the deps, including pip and pip-tools. For incremental changes, that's something i could definitely look to push here so you all could review it instead of this massive file dump.

Reply:
I should mention this did break pip-tools, so there is some merit to this version compatibility. i needed to update pip-tools to 5.4.0 (it used to be 5.1.2) to avoid compile issues. I have no idea how FoG avoided that.

Reply:
ah, ok so that was the reason of pip-tools ver bump. To be updated to 6.x something probably to match the version table. But later.